dawn/main/MenuTemplate/DawnMenu.js

```diff
@@ -14,6 +15,14 @@ const DawnMenu = {
         app.quit();
       },
     },
+    {
+      label: 'Field Control Mode',
+      click() {
+        RendererBridge.reduxDispatch({
+          type: 'TOGGLE_FIELD_CONTROL',
```

Comment:
Copying from Slack for future reference. This toggle is public; it becomes very obvious that Dawn is set to Field Control mode when toggling. Otherwise, it acts much the same was as regular Dawn, only that it may be overridden if it is connected to an active FC master station. Perhaps later it could be made into a "hidden" feature, but I don't really think that to be necessary.